Dashboard/src/components/Osteppy.js

```diff
@@ -3,24 +3,66 @@ import Panel from './common/Panel';
 import Container from './common/Container';
 import slack from './assets/slack.svg';
 import ReactMarkdown from 'react-markdown';
+import Moment from 'moment';
+
 const COMPONENT_NAME = "osteppy";
 
+// Reformat Slack messages based on:
+// https://api.slack.com/messaging/composing/formatting#retrieving-messages
+function reformatSlackMentions(text) {
+  return text.replace(/<(.*?)>/g, (match, p1) => {
+    switch(p1[0]) {
+      case '@': // content starting with `@U` or `@W` is a user mention
```

Comment:
@scrust Do you mind applying it to the channels as well? `#channel` turns into `<#C64TOPQHJ|channel` (sorry didn't mention in the issue).

One issue with just stripping the tags away is that sometimes the `ID` returned is not equal to `@username`. I think this should work for now though

Reply:
Can definitely add channel parsing, no problem!

The `ID` isn't usually equal to the `@username`, is it? E.g. I'm `@scrust` but my `ID` is `stuart.crust`; `stuart.crust` is what appears in red for me:

<img width="414" alt="image" src="https://user-images.githubusercontent.com/21987387/58504881-9dbd1d80-8159-11e9-8dc3-889d075eaaf7.png">

I figured making the names consistent, at least on the dashboard, would be the least confusing.

Reply:
For some users slack id is quite different from what appears with `@username`.
I think we will keep the `ID`s for consistency for now.